### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by szepeviktor**
### 📦 Pull Request

Teeny-tiny fixes only.

### ✅ Fixed Issues

- 🛩️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,15 +120,17 @@ Or to run an individual test file:
 ./vendor/bin/phpunit tests/MagicTest.php
 ```
 
-The library uses [PHP CS Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) for code formatting. Code must be formatted before PRs are submitted. Run the formatter with:
+The library uses [PHP CS Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) for code formatting.
+Code must be formatted before PRs are submitted. Run the formatter with:
 
 ```bash
-php-cs-fixer fix -v --using-cache=no .
+./vendor/bin/php-cs-fixer fix -v --using-cache=no .
```
Comment (szepeviktor) [inline README.md:127]: I see!! `friendsofphp/php-cs-fixer` is not required.

Could we add it? Or mention that this tool must be installed globally and must on `$PATH`.

Comment (brianrlewis) [reply README.md:127]: Sure, I think global installation instructions would be an improvement. Perhaps something like this from [here](https://cs.symfony.com/doc/installation.html)?

```
composer global require friendsofphp/php-cs-fixer
export PATH="$PATH:$HOME/.composer/vendor/bin"
```

Comment (brianrlewis) [reply README.md:127]: BTW I will tag a new release tomorrow.

Comment (szepeviktor) [reply README.md]: Linked. 🍏

**Comment by szepeviktor** [comment]
💡 Preview from the next PR.

kép

**Comment by szepeviktor** [comment]
Thank you.

**Pull request merged**